lib/route.js

```diff
@@ -572,6 +574,38 @@ function runPreParsing (err, request, reply) {
   }
 }
 
+function buildRouterOptions (options, defaultOptions) {
+  const routerOptions = options.routerOptions || Object.create(null)
+
+  const routerKeys = [
```

Comment:
Place the `const` outside of the function.
It shouldn't be recreate everytime using the `buildRouterOptions`.

Reply:
Good call out, this should be fixed now